### DISCUSSION TITLE
🌌 Night for Zotero 7 ALPHA 0.1.0-2

### DISCUSSION THREAD

**Discussion by tefkah** [tefkah/zotero-night#151] [Announcements]
# ⚠️ THIS VERSION ONLY WORKS WITH ZOTERO 7

### Dark mode epub reader
Zotero added an epub reader. This release also makes that dark mode. 
>[!NOTE]
>You can't change the colors dynamically like the pdf reader, that will happen later

<img width="1209" alt="image" src="https://github.com/tefkah/zotero-night/assets/21983833/14d201dc-c3d1-4a19-a879-1d257ff9c297">

### Dark mode pop up view
In the pdf reader, when you hover over an internal link, a lil popup pops up. This was not dark before. It is now.
It's not great tho, the border should change with the color of the page but does not at the moment

<img width="740" alt="image" src="https://github.com/tefkah/zotero-night/assets/21983833/01a8c63d-c2c3-4925-923e-2cb7a416879b">

### PDF style consistency improvements

It seems that a recent update from Zotero changed what I need to do in order to attach styles to the reader, which lead to things like #148. This should be better now.

## Issues fixed
- #148 
- #145 

## Commits
* chore: mess with the versioning (155e152)
* fix: apply filter after waiting (075d41a)
* feat: use bun (ce7aad7)
* feat: add support for epub reader (e44527b)
* fix: allow pdf portals to be styled but do it quite badly (7134f33)
* feat: allow styles to be added to more docs (467bb09)
* chore(README.md): update download link to the latest version of the plugin (2ff091a)
* chore: update update.json (43400a0)

<hr /><em>This discussion was created from the release <a href='https://github.com/tefkah/zotero-night/releases/tag/0.1.0-2'>🌌 Night for Zotero 7 ALPHA 0.1.0-2</a>.</em>

**Comment by AtoshDustosh** [comment]
There is a problem with this dark mode. 
Although the dark colour for the UI looks good, all the highlights blend into the background colour, becoming indistinguishable.
So do the words in the document outline. I can barely recognize anything.

**Reply by tefkah** [reply]
outline will be fixed in the next release, have not found a way around the highlights though. They are baked into the pdf, and i can only apply a filter on the pdf globally, which fill affect the background, text, images, and highlights in the same way.

**Comment by ianTevesAcc** [comment]
Can't wait for this ^^

Current bugs / Lacking:
- Dark mode applied to PDF's / Documents
- Dynamic switching from light mode to dark mode
- Plug In currently doesn't overpower the original Zotero Themes leading to UI bugs

Once these are fixed and implemented
This would make using Zotero amazing ^^

**Comment by simbru** [comment]
Just found this. Makes PDFs far more readable with the blueish background rather than pitch black. Some UI bugs, for example one must double click the darkmode icon causing Zotero to resize in Windows 11. Is development continuing?
